docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -59,4 +60,84 @@ let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(wi
 ```
 
 ### Using the Vehicle's Icon
-If you want to build your own lock screen view controller, it is recommended that you subclass `SDLLockScreenViewController` and use the public `appIcon`, `vehicleIcon`, and `backgroundColor` properties.
\ No newline at end of file
+If you want to build your own lock screen view controller, it is recommended that you subclass `SDLLockScreenViewController` and use the public `appIcon`, `vehicleIcon`, and `backgroundColor` properties.
+!@
+
+@![android, javaSE, javaEE] `// TODO, Android and iOS guides on this section are very different should they be aligned better?`
+# Adding the Lock Screen
+
+In order for your SDL application to be certified with most OEMs you will be required to implement a lock screen on the mobile device. The lock screen will disable user interactions with the application on the mobile device while they are using the head-unit to control application functionality. OEMs may choose to send their logo for your app's lock screen to use; the `LockScreenManager` takes care of this automatically using the default layout.
+
+!!! NOTE
+This guide assumes that you have an SDL Service implemented as defined in the [Getting Started](Getting Started With Android/Installation) guide.
+!!!
+
+There is a manager called the `LockScreenManager` that is accessed through the `SdlManager` that handles much of the logic for you. If you have implemented the `SdlManager` and have defined the `SDLLockScreenActivity` in your manifest but have not defined any lock screen configuration, you are already have a working default configuration. This guide will go over specific configurations you are able to implement using the `LockScreenManager` functionality.
+
+## Lock Screen Activity
+
+You must declare the `SDLLockScreenActivity` in your manifest. To do so, simply add the following to your app's `AndroidManifest.xml` if you have not already done so:
+
+```xml
+<activity android:name="com.smartdevicelink.managers.lockscreen.SDLLockScreenActivity"
+                  android:launchMode="singleTop"/>
+```
+
+!!! MUST
+This manifest entry must be added for the lock screen feature to work.
+!!!
+
+## Configurations
+
+The default configurations should work for most app developers and is simple to get up and and running. However, it is easy to perform deeper configurations to the lock screen for your app. Below are the options that are available to customize your lock screen which builds on top of the logic already implemented in the `LockScreenManager`.
+
+There is a setter in the `SdlManager.Builder` that allows you to set a `LockScreenConfig` by calling `builder.setLockScreenConfig(lockScreenConfig)`. The following options are available to be configured with the`LockScreenConfig`.
+
+In order to to use these features, create a `LockScreenConfig` object and set it using `SdlManager.Builder` before you build `SdlManager`.
+
+#### Custom Background Color
```

Comment:
You can use the same header for iOS and Android

Reply:
fixed